scripts/publish-package.js

```diff
@@ -3,23 +3,23 @@ const { setConfig, getFullObject, requestSuiFromFaucet } = require('./utils');
 const { Ed25519Keypair } = require('@mysten/sui.js/keypairs/ed25519');
 const { TransactionBlock } = require('@mysten/sui.js/transactions');
 const { SuiClient } = require('@mysten/sui.js/client');
+const { toB64 } = require('@mysten/sui.js/utils');
 const { execSync } = require('child_process');
 const { parseEnv } = require('./utils');
 const tmp = require('tmp');
 const fs = require('fs');
 const path = require('path');
 
-async function publishPackage(packageName, client, keypair) {
+async function getContractBuild(packageName) {
     const toml = fs.readFileSync(`${__dirname}/../move/${packageName}/Move.toml`, 'utf8');
     fs.writeFileSync(`${__dirname}/../move/${packageName}/Move.toml`, fillAddresses(toml, '0x0', packageName));
 
     // remove all controlled temporary objects on process exit
-    const address = keypair.getPublicKey().toSuiAddress();
     tmp.setGracefulCleanup();
 
     const tmpobj = tmp.dirSync({ unsafeCleanup: true });
 
-    const { modules, dependencies } = JSON.parse(
+    return JSON.parse(
```

Comment:
avoid returning a complex expression, instead assign it to a variable so it makes more sense, and return that